### DISCUSSION TITLE
Allow access control for tabs

### DISCUSSION THREAD

**Discussion by wrobson-lllow** [payloadcms/payload#2620] [Feature Requests & Ideas]
I'm currently trying to hide an entire tab to certain users depending on their role. Currently, I have only been able to hide specific fields within the tab using field access control and admin condition.

**Comment by DanRibbens** [comment]
Hi @wrobson-lllow!

I think we need to accomodate for two scenarios here:
1. A user has no access to any fields within a tab (we should hide the entire tab or possibly have a disabled state for it).
2. A tab with a name should respect the `access` defined on it, currently I'm seeing a bug with this, where access can be defined at the tab level, but is not respected.

If your tab doesn't have a `name` defined on it, your only option is to do #1.
With tab that has a name, you should be able to have #1 or #2.

**Comment by DanRibbens** [comment]
Related: https://github.com/payloadcms/payload/discussions/1840
